src/main/java/com/duckblade/osrs/toa/TombsOfAmascutConfig.java

```diff
@@ -394,6 +396,62 @@ default int chestAudioVolume()
 		return 100;
 	}
 
+	@ConfigSection(
+		name = "Reward Sarcophagus",
+		description = "Configuration for the reward sarcophagus.",
+		position = 700,
+		closedByDefault = true
+	)
+	String SECTION_REWARD_SARCOPHAGUS = "rewardSarcophagusSection";
+
+	@ConfigItem(
+		name = "Recolor white",
+		description = "Recolor the sarcophagus white lighting.",
+		position = 701,
+		keyName = "sarcophagusRecolorWhite",
+		section = SECTION_REWARD_SARCOPHAGUS
+	)
+	default boolean sarcophagusRecolorWhite()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		name = "Color",
+		description = "Color to replace sarcophagus white lighting with.",
+		position = 702,
+		keyName = "sarcophagusWhiteRecolor",
+		section = SECTION_REWARD_SARCOPHAGUS
+	)
+	default Color sarcophagusWhiteRecolor()
+	{
+		return Color.BLACK;
+	}
+
+	@ConfigItem(
+		name = "Recolor purple",
+		description = "Recolor the sarcophagus purple lighting.",
+		position = 703,
+		keyName = "sarcophagusRecolorPurple",
+		section = SECTION_REWARD_SARCOPHAGUS
+	)
+	default RecolorMode sarcophagusRecolorPurple()
+	{
+		return RecolorMode.OFF;
+	}
```

Comment:
Rather than a 3-way enable, you should allow players to rename personal purples and others' purples separate colours.